EIPS/eip-5604.md

```diff
@@ -0,0 +1,63 @@
+---
+eip: 5604
+title: NFT Lien
+description: Extend NFT to support lien
+author: Zainan Victor Zhou (@xinbenlv)
+discussions-to: https://ethereum-magicians.org/t/creating-a-new-erc-proposal-for-nft-lien/10683
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-09-05
+requires: 721
+---
+
+## Abstract
+
+This EIP introduce NFT lien, a form of security interest over an item of property to secure the recovery of liability or performance of some other obligation. It introduces interface to place and remove a lien, plus an event.
+
+## Motivation
+
+Lien is widely used for finance use cases, such as car and property liens. An use case for NFT is deed.
+The EIP provides an interface to implement interface that perform the lien holding relationships.
+
+## Specification
+
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.
+
+1. Any compliant contract MUST implement [EIP-721](./eip-721.md), [EIP-165](./eip-165.md).
+
+2. Any compliant contract MUST implement the following interface:
+
+```solidity
+interface IERC_LIEN is EIP721, EIP165 {
+    event OnLienPlaced(uint256 tokenId, address holder, bytes calldata extraParams)
+    function addLienHolder(uint256 tokenId, address holder, bytes calldata extraParams);
+    function removeLienHolder(uint256 tokenId, address holder, bytes calldata extraParams);
+}
+```
+
+3. `addLienHolder` MUST throw error if
```

Comment:
Missing end of sentence

Reply:
Thanks for catching! Will follow up. this was due to keystroke mistake